src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/AddTransparentDataEncryptionCertificateTests.ps1

```diff
@@ -0,0 +1,270 @@
+$password = "AE"
+$location = "WestCentralUS"
+$exceptionHeader = "##### Following exception occurred causing test failure #####"
+$mangedInstanceRg = "MlAndzic_RG"
+$managedInstanceName = "midemoinstance"
+
+function Test-AddTdeCertificateForSqlServerDefaultParameterSetNoPassword
+{
+	# Setup
+	$rg = Create-ResourceGroupForTest
+	$server = Create-ServerForTest $rg $location
+	try
+	{
+		Add-AzureRmSqlServerTransparentDataEncryptionCertificate `
+			-ResourceGroupName $server.ResourceGroupName `
+			-ServerName $server.ServerName `
+			-PrivateBlob $samplePrivateBlobNoPassword
+	}
+	Catch{
+		Write-Debug $exceptionHeader
+		Write-Debug  $_.Exception.Message
+		#throw if we catch an exception
+		Assert-AreEqual $true $false
+	}
+	finally
+	{
+		# Cleanup
+		Remove-ResourceGroupForTest $rg
+	}
+}
+
+<#
+	.SYNOPSIS
+	Tests adding TDE certificates to sql server with default parameter set and password
+#>
+function Test-AddTdeCertificateForSqlServerDefaultParameterSetWithPassword
+{
+	# Setup
+	$rg = Create-ResourceGroupForTest
+	$server = Create-ServerForTest $rg $location
+	try
+	{
+		Add-AzureRmSqlServerTransparentDataEncryptionCertificate `
+			-ResourceGroupName $server.ResourceGroupName `
+			-ServerName $server.ServerName `
+			-PrivateBlob $samplePrivateBlobWithPassword `
+			-Password $password
+	}
+	Catch{
+		Write-Debug $exceptionHeader
+		Write-Debug  $_.Exception.Message
+		#throw if we catch an exception
+		Assert-AreEqual $true $false
+	}
+	finally
+	{
+		# Cleanup
+		Remove-ResourceGroupForTest $rg
+	}
+}
+
+<#
+	.SYNOPSIS
+	Tests adding TDE certificates to managed instance with default parameter set and no password
+#>
+function Test-AddTdeCertificateForManagedInstanceDefaultParameterSetNoPassword
+{
+	try
```

Comment:
@nivimsft this test doesn't have any resource construction like the tests above

Reply:
For these tests existing resources are being used since automatic provisioning of managed instances is not possible at this point. These tests are used only for manual validation and are marked as skipped